paddle/operators/roi_pool_op.cu

```diff
@@ -0,0 +1,265 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/platform/cuda_helper.h"
+#include "paddle/operators/roi_pool_op.h"
+
+namespace paddle {
+namespace operators {
+
+#define FLT_MAX __FLT_MAX__
+
+constexpr int PADDLE_OPERATORS_ROIPOOL_CUDA_NUM_THREADS = 512;
```

Comment:
According to https://google.github.io/styleguide/cppguide.html#Constant_Names, constants should be named as

```c++
constexpr int kPaddleOperatorsROIPoolNumCUDAThreads = 512;
```

Also, I vaguely remember (but not sure) that we can declare this variable *static* so to limit its usage within this file, and we can have a much shorter name for it:

```c++
static constexpr int kNumCUDAThreads = 512;
```